en/django_start_project/README.md

```diff
@@ -199,7 +199,7 @@ Congratulations! You've just created your first website and run it using a web s
 
 ![Install worked!](images/install_worked.png)
 
-While the web server is running, you won't see a new command-line prompt to enter additional commands. The terminal will accept new text but will not execute new commands. This is because the web server continuously runs in order to listen for incoming requests.
+Note that a command window can only run one thing at a time, and the command window you opened earlier is running the web server. So, as long as the web server is running and waiting for additional incoming requests, you won't see a new command-line prompt to enter additional commands. The terminal will accept new text but will not execute new commands.
```

Comment:
I think the `So, ` can be dropped here. I also think the "you won't see a new command line prompt" might be confusing. I think it's sufficient to state it won't execute new commands but only echo them.